ROADMAP.md

```diff
@@ -10,66 +10,75 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
-    * Resizing volumes [#1169](https://github.com/rook/rook/issues/1169)
-  * Improved data placement and pool configuration (CRUSH maps) [#560](https://github.com/rook/rook/issues/560)
-  * OSDs
-    * Run on arbitrary PVs (local storage) as an alternative to host path [#919](https://github.com/rook/rook/issues/919)
-    * Detect new nodes for OSD configuration [#2208](https://github.com/rook/rook/issues/2208)
-    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
-  * File
-    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
-    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
-  * Object
-    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
-  * Mgr and plugins
-    * Placement group balancer support (enable the mgr module)
+  * Add support for Ceph Nautilus
+  * Beta release of CSI plug-in [#1385](https://github.com/rook/rook/issues/1385)
+    * Dynamic volume resizing [#1169](https://github.com/rook/rook/issues/1169)
+  * Automatically detect new nodes [#2208](https://github.com/rook/rook/issues/2208)
+  * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
+  * Create NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
+  * Document metadata backup and disaster recovery [#592](https://github.com/rook/rook/issues/592)
 * CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
   * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
 * EdgeFS
   * Declare EdgeFS CRDs to be Beta v1 [#2506](https://github.com/rook/rook/issues/2506)
   * Automatic host validation [#2409](https://github.com/rook/rook/issues/2409)
-  * Target VDEVs
-    * Support for flexible write I/O “sync” option [#2367](https://github.com/rook/rook/issues/2367)
+  * Allow for more performance vs. availability tradeoff configuration [#2367](https://github.com/rook/rook/issues/2367)
   * Object
     * OpenStack/SWIFT CRD [#2509](https://github.com/rook/rook/issues/2509)
     * Support for S3 bucket as DNS subdomain [#2510](https://github.com/rook/rook/issues/2510)
-  * Block (iSCSI)
-    * Support Block CSI [#2507](https://github.com/rook/rook/issues/2507)
-  * Mgr
-    * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
-    * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Support Block (iSCSI) CSI [#2507](https://github.com/rook/rook/issues/2507)
+  * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
+  * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
 * Minio
   * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+
+
+## Initial planning for Rook 1.1
```

Comment:
These are items that are more or less committed for 1.1, right? Items that we'd like to do soon, but don't have an owner would be put in the "Future Improvements" section? There are some items here that I was thinking would be moved to the future improvements such as the integration testing improvements and a few of the finer details.

Reply:
That sounds best, I agree.

Reply:
I added any item from the v1.1 section without an assignee to the "Future Improvements" section. This means now that CockroachDB, Minio, and EdgeFS don't have any v1.1 milestones planned at this time.